webpack.config.js

```diff
@@ -35,95 +34,56 @@ const LIBRARY_BUNDLE_CONFIG = env => ({
 
   output: {
     // Generate the bundle in dist folder
-    path: resolve('./dist'),
-    filename: 'index.js',
-    library: 'kepler.gl'
-    // libraryTarget: 'umd'
+    path: resolve('./umd'),
+    filename: 'keplergl.js',
+    globalObject: 'this',
```

Comment:
I have had to include this before due to webworkers breaking in with window/self not defined in the workerfile since of how it gets bundled. Why the inclusion of `globalObject: 'this'` here?

Reply:
Without that option. keplerGl will not be injected in the global scope correctly. 

Error without globalObject: 'this',:
```
Uncaught ReferenceError: KeplerGl is not defined
```